.vib/moodle/cypress/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
+  "baseUrl": "http://localhost",
+  "pageLoadTimeout": 150000,
+  "defaultCommandTimeout": 50000,
+  "chromeWebSecurity": false,
```

Comment:
Just being curious, are all these params necessary? I don't recall Moodle being particularly resource-heavy.

Reply:
@mdhont Could you verify the `chromeWebSecurity` has to be disabled? By default, there shouldn't be any need to modify its default value.

Reply:
Yes, this is necessary due to this error:

```
  Blocked a frame with origin &quot;http://xxxxxxxxxxxxxxx.com&quot; from accessing a cross-origin frame.

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

```